### ISSUE TITLE
Not implemented on this Platform:Error

### ISSUE THREAD

**Issue by AnishZz** [python-escpos/python-escpos#378]
Hi, This is Anish. I have implemented your 'Python-escpos' library as in the pypi.org. It
works like charm in Linux machines whereas, in windows, it shows 'NotImplementedError: Operation not supported or unimplemented on this platform'.
Can anybody post a detailed instruction of
having solved the same issue .

Printer: Every Com 58

python version: 3.7 (64 bit)

operating system: windows 10 (64 bit)
 It would be great to hear back from anyone.

**Comment by patkan** [comment]
Could you please state, which version of the software you are using? (Information from pip or calling `python-escpos version` or importing python-escpos and checking `__version__`)

**Comment by AnishZz** [comment]
Hi Pat, good to hear from you.
We installed from pypi.org via pip installer. and we installed the latest one mentioned as 2.2.0.
It worked excellent when we used it with our 25 linux sysytems, where as, at the other end, clients had 1000 windows systems for solution deployment. where, it showed previously mentioned error.
Then we from our end tried with some of the windows systems, it showed the same error. So, it would be a greatest help for us if this error is resolved. Waiting to hearback

Thanks and Regards,
Anish
